src/monopoly.js

```diff
@@ -1,48 +1,158 @@
-export class Monopoly {
+import BoardState from "./boardState";
+import Player from "./player";
+import fs from 'fs';
+import events from 'events';
+
+export default class Monopoly {
   constructor(){
-    this.PropertyList = null;
+    this.PropertyArray = null;
     this.ChanceList = null;
     this.CommChestList = null;
-    this.PlayerList = null;
+    this.PlayerArray = new Array(0);
+    this.currentPlayer = null;
+    this.Emitter = new events();
   }
 
-  loadPlayer(){}
+  loadPlayer(name, regID, socketID){
+    const regArray = JSON.parse(fs.readFileSync('./dist/regIDArray.json'));
+    //TODO: Check for registered ID
+    if(regArray.indexOf(regID) >= 0){
+      this.PlayerArray.push(new Player(name, regID, socketID));
+      console.log('Player ' + this.PlayerArray[this.PlayerArray.length - 1].person + ' added.');
+      if(regArray.length == this.PlayerArray.length){
+        console.log('runGame');
+        this.runGame();
+        return this;
```

Comment:
You can get rid of these two `return this;` since they'll both fall down into that last one.